### ISSUE TITLE
Error when calling nanoid in browser (from 5.0.5)

### ISSUE THREAD

**Issue by TomPradat** [ai/nanoid#470]
When calling nanoid from the browser, I get the following error in the console :

![image](https://github.com/ai/nanoid/assets/16164512/6a1dc0b7-1e90-450c-b73e-dbb581393fd2)

Here is a minimal repro :

```javascript
import { nanoid } from "nanoid";

document.getElementById("app").innerHTML = `
<h1>Hello : ${nanoid()} </h1>
`;
```

https://codesandbox.io/p/devbox/tender-perlman-forked-9c35qx?embed=1&file=%2Fsrc%2Findex.mjs&showConsole=true

> NB : If the error doesn't appear in the embed page console, open the main console

**Comment by lkaisers** [comment]
I just experienced the same issue. In my case it is inside a React app any only in watch mode (using Parcel).

**Comment by ai** [comment]
Ask Parcel team. Seems like there is something wrong with `import` support.

The example works fine in Vite or webpack.

**Comment by WilhelmYakunin** [comment]
Developers,

the Aprcel delete ALL unused exports and import sduring tree-shaking. This indicated in [documention](https://parceljs.org/features/scope-hoisting/#how-scope-hoisting-works)

This behaviour cased of deleting variable of urlAlphabet which actually is in use in geneating nanoid.
Please visit this lines 
<code>https://github.com/ai/nanoid/blob/main/index.browser.js#L4</code> 
<code>https://github.com/ai/nanoid/blob/main/index.browser.js#L6</code> 
<code>https://github.com/ai/nanoid/blob/main/index.browser.js#L59</code>

So because the urlAlphabet is not in use the Parcel loses [the file](https://github.com/ai/nanoid/blob/main/url-alphabet/index.js) acronim name in bundle. Actually it seems as error during a load. I can show more detailes if you want to. But [here](https://codesandbox.io/p/devbox/tender-perlman-forked-gmxshl?file=%2Fsrc%2Findex.mjs) I expilicetly use the 'urlAlphabet' variable and the nanoid is generated as expected.

However, I tend to see a problem of the common agreements betweeen the delovepers. Same name for imported variale while it is aslo exported in the same file is not forbiden in JS. But it can misconfused while static analisize is on.  

@ai do you mind if create pull from my [repo](https://github.com/WilhelmYakunin/nanoid)?

**Comment by ai** [comment]
@WilhelmYakunin fixed the issue in 5.0.7.

**Comment by louh** [comment]
Hi all,
I've been facing this exact error since 5.0.5, with a Parcel build, and it still isn't fixed for me in 5.0.7. However, I'm a little stumped because my build works just fine locally _and_ in a production environment (even before 5.0.7), but it _only_ fails when building in a Cypress testing environment. I'm not seeing any differences in my configuration that would possibly be forcing Parcel to improperly or aggressively tree-shake in this one environment, but I might be missing something. Might there be anyone out there that has faced this same issue as well?
